src/AcceptanceTests/NServiceBus.Encryption.MessageProperty.AcceptanceTests.csproj

```diff
@@ -32,6 +32,18 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="NServiceBus.AcceptanceTesting, Version=6.0.0.0, Culture=neutral, PublicKeyToken=9fc386479f8a226c, processorArchitecture=MSIL">
+      <HintPath>..\packages\NServiceBus.AcceptanceTesting.6.2.0-alpha0166\lib\net452\NServiceBus.AcceptanceTesting.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="NServiceBus.Core, Version=6.0.0.0, Culture=neutral, PublicKeyToken=9fc386479f8a226c, processorArchitecture=MSIL">
+      <HintPath>..\packages\NServiceBus.6.2.0-alpha0166\lib\net452\NServiceBus.Core.dll</HintPath>
```

Comment:
took the latest 6.2 unstable since it contains a few improvements on the acceptance testing framework (and once we want to test duplicate encryption configuration we need 6.2 too).